src/components/players/detailsPlaceholder.module.scss

```diff
@@ -0,0 +1,58 @@
+@import 'src/scss/variables.scss';
+
+.playerDetailsContainer {
```

Comment:
should be 

`.playerDetailsContainer {`

Reply:
Looks like you want me to rename it container later on down.